Extension/ASPNET-MVC-Extension/Project-Conversion.md

```diff
@@ -49,11 +49,11 @@ The following steps help you use the Syncfusion Project Conversion in the existi
    **Option 2:**  
    Right-click the Project from Solution Explorer, select **Syncfusion Essential JS 1**, and choose **Convert to Syncfusion MVC (Essential JS 1) Application...** Refer to the following screenshot for more information.
 
-   ![Syncfusion Essential JS 1 ASP.NET MVC Web Project Conversion add-in](Convert-into-Syncfusion-MVC-project_images/ProjectConversion-img1.png)
+   ![Syncfusion Essential JS 1 ASP.NET MVC Extension in Web Project Conversion add-in](convert-into-syncfusion-mvc-project-images/aspnet-mvc-extension-in-project-conversion-in-web-add-in.png)
```

Comment:
You replaced the Folder name (_) to hyphen. Please check and change in all places.

Reply:
Resolved the comment issue